### ISSUE TITLE
Language selector appearance (icon to text alignment)

### ISSUE THREAD

**Issue by hwndmaster** [crow-translate/crow-translate#407]
There are two sides of language selectors on the top of the main window: The Source language selector and The Target language selector.

The left part of the selector looks fine, it is easy to distinguish which text is related to which flag and intuitively you click an appropriate button.
In contrary, the right side has a different icon alignment in buttons and it makes some confusion because the text is closer an icon of the button nearby rather than to its own icon.

Like on the example below (screenshot taken from readme.md) when a user wants to select German as the target language, a user may mistakenly click the button of "English" language, because intuitively its text is closest to the icon of desired language.

![image](https://user-images.githubusercontent.com/5339929/161959015-ad81d6de-df3d-4484-ae55-5d8fb05f8cc0.png)

My proposition is to make the button alignment (Icon + Text) on the right side to be looking in the same way as on the left side (arrows shown on the screenshot).

Alternatively, make it configurable in the settings.

**Comment by Shatur** [comment]
I agree with the idea. It happens because I enabled right-to-left layout display for the right sidebar... Maybe there is an option show icons on the right independently.
